cpp/src/phonenumbers/phonenumbermatcher.cc

```diff
@@ -421,12 +422,26 @@ PhoneNumberMatcher::PhoneNumberMatcher(const string& text,
       max_tries_(numeric_limits<int>::max()),
       state_(NOT_READY),
       last_match_(NULL),
-      search_index_(0) {
+      search_index_(0),
+      isInputInCleanUTF8_(true) {
+  isInputInCleanUTF8_ =  CheckInputUTF8OrNot();
 }
 
+
 PhoneNumberMatcher::~PhoneNumberMatcher() {
 }
 
+// static
+bool PhoneNumberMatcher::CheckInputUTF8OrNot() {
```

Comment:
Better method name is just IsInputUtf8 (rather than "or not" at the end)

Reply:
Done. +1